roles/nuage_node/tasks/main.yaml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Install Nuage VRS
+  sudo: true
+  yum: name={{ vrs_rpm }} state=present
+  
+- name: Set the uplink interface 
+  sudo: true
+  lineinfile: dest={{ vrs_config }} regexp=^NETWORK_UPLINK_INTF line='NETWORK_UPLINK_INTF={{ uplink_interface }}'
+
+- name: Set the Active Controller 
+  sudo: true
+  lineinfile: dest={{ vrs_config }} regexp=^ACTIVE_CONTROLLER line='ACTIVE_CONTROLLER={{ vsc_active_ip }}'
+
+- name: Set the Standby Controller 
+  sudo: true
+  lineinfile: dest={{ vrs_config }} regexp=^STANDBY_CONTROLLER line='STANDBY_CONTROLLER={{ vsc_standby_ip }}'
+  when: vsc_standby_ip is defined
+
+- name: Install plugin rpm
+  sudo: true
+  yum: name={{ plugin_rpm }} state=present
+
+- name: Set the client cert
+  sudo: true
+  lineinfile: dest={{ vsp_k8s_yaml }} regexp=^clientCert line='clientCert{{':'}} {{ client_cert }}'
+
+- name: Set the client key 
+  sudo: true
+  lineinfile: dest={{ vsp_k8s_yaml }} regexp=^clientKey line='clientKey{{':'}} {{ client_key }}'
+
+- name: Set the CACert 
+  sudo: true
+  lineinfile: dest={{ vsp_k8s_yaml }} regexp=^CACert line='CACert{{':'}} {{ ca_cert }}'
+
+- name: Set the enterprise
+  sudo: true
+  lineinfile: dest={{ vsp_k8s_yaml }} regexp=^enterpriseName line='enterpriseName{{':'}} {{ enterprise }}'
+
+- name: Set the domain
+  sudo: true
+  lineinfile: dest={{ vsp_k8s_yaml }} regexp=^domainName line='domainName{{':'}} {{ domain }}'
+
+- name: Set the master API server 
+  sudo: true
+  lineinfile: dest={{ vsp_k8s_yaml }} regexp=^masterApiServer line='masterApiServer{{':'}} {{ api_server }}'
+
+- name: Set the docker bridge name 
+  sudo: true
+  lineinfile: dest={{ vsp_k8s_yaml }} regexp=^dockerBridgeName line='dockerBridgeName{{':'}} {{ docker_bridge }}'
+  notify:
+    - restart vrs
+    - restart node 
```

Comment:
Is the node restart here simply to "pick up the underlying network changes"? Or is there deeper integration with the kubelet than appears from the tasks listed here?